games/pirate-jam-2024/src/model/activations/card/cardActivations.ts

```diff
@@ -103,6 +108,7 @@ export class DecomposeCardActivation
   implements AsyncCardActivation, DynamicDescriptionHolder
 {
   async activateTargettedAsync(context: ActivationContextSingleCard) {
+    context.targetCard.view.setVisible(false)
```

Comment:
@DrParanoia are you sure this is the right place?

Reply:
@kibertoad Pretty much. This is a `Decompose` activation, which means, the card will be destroyed.

`AnimateCardActivation` does not hide the card anymore, it only plays animations on top of the card (for example attack animations, which do not require the card to be hidden).

So if we don't hide the card here, then the animation will be played first, and only after the card will disappear, which does not look right.